modules/apps/foundation/address-formatter/address-formatter-api/src/main/java/com/liferay/address/formatter/BaseAddressFormatter.java

```diff
@@ -37,7 +43,7 @@ public String getCountry(Address address) {
 		Country country = countryService.fetchCountry(countryId);
 
 		if (country != null) {
-			return country.getName();
+			return country.getName(getUserLocale());
```

Comment:
If I understand this correctly, what we are doing here is using the user's locale, but only if address doesn't have a country or it's unknown. Is that correct?

If so, it's not aligned with the Story description. Which is wrong?